cascade-cli/src/Cascade/CLI/Data/Model/Config.hs

```diff
@@ -65,14 +81,21 @@ deriving stock instance Show Final
 deriving via Generically Partial instance Semigroup Partial
 deriving via Generically Partial instance Monoid Partial
 
-data Error = BusyHttpPortError
+data Error = BusyHttpPortError Word16
   deriving stock (Show, Eq)
 
 type Errors = NonEmpty Error
 
+prettyPrintError :: Error -> Text
+prettyPrintError (BusyHttpPortError port) = "Port " <> show port <> " is busy, try another port"
```

Comment:
```diff
- Port XXX is busy, try another port
+ Port XXX is busy, try another port.
```

Reply:
Also, let's move the `prettyPrintError` as close as possible to the `Error` datatype.